### PR TITLE
Add background playback and compact notification controls

### PR DESCRIPTION
**Pull request by dkanada**
should be merged after #73 but can be reviewed now

### CODE REVIEW

File: src/NativeShell/plugin.xml [hunk 1]
```diff
@@ -13,27 +13,24 @@
...
-    <source-file src="src/logging/LogFileFilter.java" target-dir="src/com/mb/android/logging" />
-    <source-file src="src/media/RemotePlayerService.java" target-dir="src/com/mb/android/media" />
+    <source-file src="src/NativeShell.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/AppLogger.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/LogbackLogger.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/RemotePlayerService.java" target-dir="src/org/jellyfin/mobile" />
+    <source-file src="src/CustomWebViewClient.java" target-dir="src/org/jellyfin/mobile" />
     <framework src="com.android.support:support-v4:23.3.0" />
-    <framework src="com.android.support:appcompat-v7:23.3.0" />
-    <framework src="com.android.support:mediarouter-v7:23.3.0" />
```
Comment (JustAMan) [inline src/NativeShell/plugin.xml]: why these two are no longer needed?

File: src/NativeShell/src/RemotePlayerService.java [hunk 2]
```diff
@@ -92,9 +89,16 @@ private void handleIntent(Intent intent) {
             switch (action) {
                 case Constants.ACTION_PLAY:
                     mediaController.getTransportControls().play();
+                    // background playback for an hour
+                    if (NativeShell.wakeLock != null && !NativeShell.wakeLock.isHeld()) {
+                        NativeShell.wakeLock.acquire(36000000);
```
Comment (JustAMan) [inline src/NativeShell/src/RemotePlayerService.java]: hmm, why limit to 1 hour background playback?.. seems rather arbitrary number...

Comment (joshuaboniface) [reply src/NativeShell/src/RemotePlayerService.java]: Agreed, I know I've casted for longer ;-)

File: src/NativeShell/plugin.xml (hunk 1 above)
Comment (dkanada) [reply src/NativeShell/plugin.xml]: They were used in the chromecast plugin which is now set up in another repository.

**Pull request merged**